pkg/util/util.go

```diff
@@ -244,3 +246,11 @@ func GetMappedGitOpsComponent(component appstudiov1alpha1.Component) gitopsgenv1
 	}
 	return gitopsMapComponent
 }
+
+// GenerateUserUniqueHash generates a unique hash from the combination of cluster name and the namespace
+// in order to not expose user's username via namespace of a particular resource potentially.
+func GenerateUserUniqueHash(clusterName string, namespace string) string {
+	h := sha256.New()
+	h.Write([]byte(clusterName + namespace))
+	return base64.URLEncoding.EncodeToString(h.Sum(nil))[0:5]
```

Comment:
Does this length enough to guarantee uniqueness?

Reply:
It keeps the same length in the change made in #203.

Reply:
I see, thanks